rust/datafusion/src/execution/context.rs

```diff
@@ -720,6 +735,9 @@ mod tests {
         ];
         assert_batches_eq!(expected, &results);
 
+        assert_eq!(ctx.deregister_table("dual"), true);
+        assert_eq!(ctx.deregister_table("dual"), false);
```

Comment:
What do you think about moving this to a separate test? This way, if something fails, it is more obvious what caused the failure.

Reply:
I'll do that. I hadn't done it originally because there is a fair amount of boilerplate to setting up a data source, but I agree that it makes sense to test is separately.